src/cli/output/installDependencies.ts

```diff
@@ -14,7 +16,24 @@ import installPackages from "../../utils/installPackages.js";
  */
 export default async function installDependencies(input: UserInput) {
   const { plugins, projectDir, appConfig } = input;
-
+   const pkgManager = getPackageManager();
+
+
+	const installDeps =
```

Comment:
This can be simplified to `const installDeps = process.env.INSTALL_DEPENDENCIES === 'true'`